Modules/DataModel/Sources/PocketCastsDataModel/Public/Model/Episode.swift

```diff
@@ -206,6 +206,43 @@ public class Episode: NSObject, BaseEpisode {
             public let url: String
             public let type: String
             public let language: String?
+
+            public var transcriptFormat: TranscriptFormat? {
+                return TranscriptFormat(rawValue: self.type)
+            }
+        }
+
+        public enum TranscriptFormat: String {
```

Comment:
What do you think about moving this outside of `Episode`?

I feel like `Episode` and `Transcript` are not **directly** related and we're adding a lot of knowledge about Transcripts inside an `Episode` (like file extension, best transcript, etc).

Maybe `TranscriptManager` is a better place?

Reply:
Are you suggesting to move `TranscriptFormat` only right?

Reply:
Yep!

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/1955/commits/fcf6f156be0a05eca84e1a1526b05e64c43e3977

Decided to move it to a separate file.